src/sdk/helpers/rect.js

```diff
@@ -6,6 +6,10 @@
  * @param height
  * @constructor
  */
+
+var _ = require('underscore');
+var jQuery = require('jquery');
```

Comment:
`jQuery` is generally required as `$`. There's nothing else using it there so it should be fine I guess if it stays as it is.